apps/shared.hu.i18n

```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
-DataNotSuitable = "Az adatok nem megfelelöek"
-DataTab = "Data"
-DefaultSetting = "Alapbeállítások"
+CopyColumnInList = "Az oszlopot egy listába másolni"
+DataNotSuitable = "Az adatok nem felelnek meg"
+DataTab = "Adatok"
+DefaultSetting = "Alapállítások"
 Deg = "deg"
 Deviation = "Varianca"
 DisplayValues = "Értékek mutatása"
 Empty = "Üres"
 Eng = "eng"
-ExitExamMode1 = "Kilépés a vizsga "
+ExitExamMode1 = "Kilépni a vizsga "
 ExitExamMode2 = "módból?"
 Exponential = "Exponenciális"
 FillWithFormula = "Töltse ki egy képlettel"
 ForbiddenValue = "Tiltott érték"
-FunctionColumn = "0 (0) oszlop"
+FunctionColumn = "0(0) oszlop"
 FunctionOptions = "Funkció opciók"
-Goto = "Menj ide"
+Goto = "Menj ..."
 GraphTab = "Grafikon"
-HardwareTestLaunch1 = "Ön elindítja a hardvert"
-HardwareTestLaunch2 = " tesztet.  A teszt végén neked"
-HardwareTestLaunch3 = "kell vissza állítani az eszközt,és"
-HardwareTestLaunch4 = "az összes adat törlödni fog."
+HardwareTestLaunch1 = "A hardverteszt indítása :"
+HardwareTestLaunch2 = "Nyomjon a reset gombra a"
+HardwareTestLaunch3 = "teszt megállításához (ez"
+HardwareTestLaunch4 = "az adatokat törölni fogja)"
 Initialization = "Elöbeállítás"
 IntervalSet = "Állítsa be az intervallumot"
 Language = "Nyelv"
-LowBattery = "Alacsony töltöttség"
+LowBattery = "Majdnem kimerült az elem"
 Mean = "középérték"
-Move = " Move: "
+Move = " Odébb rakni: "
 NameCannotStartWithNumber = "Egy név nem kezdöthet számmal"
 NameTaken = "Ez a név foglalt"
 NameTooLong = "Ez a név túl hosszú"
 Next = "következö"
+NEnd = "N vége"
 NoDataToPlot = "Nincs rajzolható adat"
 NoFunctionToDelete = "Nincs törölhetö függvény"
 NoValueToCompute = "Nincs számítható érték"
-NEnd = "N vége"
 NStart = "N kezdete"
-Ok = "Megerösítés"
+Ok = "Érvényesítés"
 Or = " vagy "
-Orthonormal = "Ortonormál"
-Plot = "Grafikon elkészítése"
+Orthonormal = "Ortonormált"
+Plot = "Grafikon rajzolása"
 PoolMemoryFull1 = "A memória megtelt."
 PoolMemoryFull2 = "Kérem próbálja újra."
 Rad = "rad"
 Rename = "Átnevezés"
-RoundAbscissa = "Egész szám"
+RoundAbscissa = "Egész abszcissza"
 Sci = "sci"
 SquareSum = "Négyzetek összege"
 StandardDeviation = "Alap eltérés"
-StoreExpressionNotAllowed = "'tárolás' nem engedélyezett"
 StatTab = "Statisztikák"
 Step = "Lépés"
 StorageMemoryFull1 = "A memória megtelt."
-StorageMemoryFull2 = "Adatok törlése és próbálkozzon újra."
+StorageMemoryFull2 = "Törlöljön adatokat és próbálkozzon újra."
+StoreExpressionNotAllowed = "'szore' nem engedélyezett"
 SyntaxError = "Szintaxis hiba"
 Sym = "sym"
 TEnd = "T vég"
 ThetaEnd = "θ vége"
 ThetaStart = "θ kezdete"
 TStart = "T kezdete"
-ToZoom = "Zoom:"
+ToZoom = "Nagyítani : "
 Trigonometric = "Trigonometrikus"
-UndefinedValue = "Nincs meghatározva érték"
+UndefinedValue = "Meghatározatlan adat"
 ValuesTab = "Táblázat"
 Warning = "Figyelem"
 XEnd = "X vége"
 XStart = "X kezdete"
-Zoom = "Zoom"
-Developers = "Fejlesztök"
+Zoom = "Nagyítás"
+Developers = "Kifejlesztök"

```

Comment:
Although we say "kifejlesztők" its generally meant for vaccines and such. "Fejlesztők" is the correct term here.